packages/web-api/src/methods.ts

```diff
@@ -634,6 +758,25 @@ export interface APITestArguments extends WebAPICallOptions {}
   /*
    * `apps.*`
    */
+export interface AppsEventAuthorizationsListArguments
+  extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  event_context: string;
+}
+export interface AppsPermissionsInfoArguments extends WebAPICallOptions, TokenOverridable {}
```

Comment:
It's totally fine to have these but just for your information, `apps.permissions.*` are for Workspace Apps. So, no new developer will be using these ones.